lisa/wa_results_collector.py

```diff
@@ -492,6 +492,48 @@ def row_filter(r): return True
 
         return ret
 
+    def _read_energy_instrument_metrics(self, path):
+        """
+        Parse the WA energy_measurement output.
+        """
+        path_cache = path + ".parquet"
+
+        if not os.path.isfile(path_cache):
+            try:
+                df = pd.read_csv(path)
```

Comment:
Could the CSV be modified in any way ?
I guess not, in which case it's fine to not checksum the CSV and just use parquet if it's there. Maybe add a comment stating that assumption

Reply:
Done